projects/aws-observability/aws-otel-collector/Makefile

```diff
@@ -0,0 +1,30 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO_OWNER=aws-observability
+REPO=aws-otel-collector
+
+BINARY_TARGET_FILES=aoc
+SOURCE_PATTERNS=./cmd/awscollector
+
+IMAGE_COMPONENT=aws-otel-collector/aws-otel-collector
+IMAGE_NAMES=aws-otel-collector
+
+HAS_HELM_CHART=true
+HELM_GIT_TAG:=$(shell cat HELM_GIT_TAG)
+HELM_SOURCE_OWNER=open-telemetry
+HELM_SOURCE_REPOSITORY=opentelemetry-helm-charts
+HELM_DIRECTORY=charts/opentelemetry-collector
+HELM_DESTINATION_REPOSITORY=aws-otel-collector/aws-otel-collector-helm
```

Comment:
if we change direction where we are using the controller, it seems like this name would be confusing. maybe no collector in the destination or adot-helm or something? maybe there is another helm chart for that?